charts/flyte/values-gcp.yaml

```diff
@@ -1,3 +1,186 @@
+userSettings:
+  googleProjectId: <GOOGLE_PROJECT_ID>
+  dbHost: <DB_HOST>
+  dbPassword: <DB_PASSWORD>
+  bucketName: <BUCKET_NAME>
+
+#
+# FLYTEADMIN
+#
+
+flyteadmin:
+  replicaCount: 2
+  serviceAccount:
+    # -- If the service account is created by you, make this false, else a new service account will be created and the iam-role-flyte will be added
+    # you can change the name of this role
+    create: true
+    annotations:
+      # Needed for gcp workload identity to function
+      # https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity
+      iam.gke.io/gcp-service-account: flyte@{{ .Values.userSettings.googleProjectId }}.iam.gserviceaccount.com
+  service:
+    annotations:
+      # Required for the ingress to properly route grpc traffic to grpc port
+      cloud.google.com/app-protocols: '{"grpc":"HTTP2"}'
+  resources:
+    limits:
+      ephemeral-storage: 200Mi
+    requests:
+      cpu: 50m
+      ephemeral-storage: 200Mi
+      memory: 200Mi
+  service:
+    annotations:
+      # Required for the ingress to properly route grpc traffic to grpc port
+      cloud.google.com/app-protocols: '{"grpc":"HTTP2"}'
+  affinity:
+    podAntiAffinity:
+      requiredDuringSchedulingIgnoredDuringExecution:
+        - labelSelector:
+            matchLabels:
+              app.kubernetes.io/name: flyteadmin
+          topologyKey: kubernetes.io/hostname
+
+#
+# DATACATALOG
+#
+
+datacatalog:
+  replicaCount: 2
+  serviceAccount:
+    # -- If the service account is created by you, make this false, else a new service account will be created and the iam-role-flyte will be added
+    # you can change the name of this role
+    create: true
+    annotations:
+      # Needed for gcp workload identity to function
+      # https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity
+      iam.gke.io/gcp-service-account: flyte@{{ .Values.userSettings.googleProjectId }}.iam.gserviceaccount.com
+  resources:
+    limits:
+      cpu: 500m
+      ephemeral-storage: 200Mi
+    requests:
+      cpu: 50m
+      ephemeral-storage: 200Mi
+      memory: 200Mi
+  service:
+    annotations:
+      # Required for the ingress to properly route grpc traffic to grpc port
+      cloud.google.com/app-protocols: '{"grpc":"HTTP2"}'
+  affinity:
+    podAntiAffinity:
+      requiredDuringSchedulingIgnoredDuringExecution:
+        - labelSelector:
+            matchLabels:
+              app.kubernetes.io/name: datacatalog
+          topologyKey: kubernetes.io/hostname
+
+#
+# FLYTEPROPELLER
+#
+
+flytepropeller:
+  replicaCount: 2
```

Comment:
Is leader election enabled by default for this?

Reply:
Ah I see that it is, per: https://github.com/flyteorg/flyte/blob/fcf5cb80b24ff34062097316d8dc6e3d8ba29327/charts/flyte/values.yaml#L599